software/wmla120_ansible/uninstall_wmla120.yml

```diff
@@ -0,0 +1,42 @@
+---
+    # source /opt/ibm/spectrumcomputing/profile.platform
+    #  egosh user logon -u Admin -x Admin
+    #  egosh service stop all
+    #  egosh ego shutdown
+    #  sudo bash /opt/ibm/spectrumcomputing/uninstall/deeplearningimpactuninstall-1.2.2.0.sh
+    #  sudo bash /opt/ibm/spectrumcomputing/uninstall/conductorsparkuninstall-2.3.0.sh
+    #  sudo rm -rf /root/.powerai
+    #  sudo rm -rf /opt/anaconda3
+    #  sudo rm -rf /opt/ibm
+- name: Gathering data 
+  set_fact:
+    install_dir_ibm: "/opt/ibm"
+    install_dir_anaconda: "/opt/anaconda3"
+    install_dir_license: "/root/.powerai"
+    stop_services: "source /opt/ibm/spectrumcomputing/profile.platform && egosh user logon -u Admin -x Admin && egosh service stop all && printf 'Y'| egosh ego shutdown"
+    uninstall_dli: "printf 'Y' | sudo bash /opt/ibm/spectrumcomputing/uninstall/deeplearningimpactuninstall-1.2.2.0.sh"
+    uninstall_conductor: "printf 'Y' | sudo bash /opt/ibm/spectrumcomputing/uninstall/conductorsparkuninstall-2.3.0.sh"
+
+- name: Stop All Spectrum Computing Services and Shutdown
+  shell: "{{ stop_services }}"
+  become: yes
+
```

Comment:
I'd like to see some additional validation and conditional execution of subsequent uninstall steps. We want to be sure we're uninstalled before trying another install. Also my experience with uninstalling WMLA is that it's unforgiving if the steps don't complete properly and in order.